pkg/network/network.go

```diff
@@ -30,6 +30,12 @@ const (
 	// included in request metrics.
 	ProbeHeaderName = "k-network-probe"
 
+	// TripHeaderName is the name of a header that can be used to track
+	// where a message go through.  Current uses are:
+	// - activator adds its name ("activator") to the requests it proxies
+	//   to the queue-proxy of a revision.
+	TripHeaderName = "k-trip"
```

Comment:
I'd go with `k-proxy-req` or along those lines.

Reply:
Done.  I've probably thought too much about spoiling the header space and tried to have multiple uses for the header.  It's internal so probably doesn't matter.